src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/AddressServiceImplTest.java

```diff
@@ -39,6 +40,8 @@
 
   @InjectMocks AddressService addressService = new AddressServiceImpl();
 
+  @Mock CaseServiceImpl caseServiceImpl = new CaseServiceImpl();
```

Comment:
now of these mocks need assignment.
Replace with:
```
  @Mock  AddressServiceClientServiceImpl addressClientService;
  @InjectMocks AddressServiceImpl addressService;
  @Mock CaseServiceImpl caseServiceImpl;
```

(note change to concrete type of `addressService`)

Reply:
Changed as suggested.